website/content/docs/configuration/server.mdx

```diff
@@ -267,6 +267,14 @@ server {
 - `job_tracked_versions` `(int: 6)` - Specifies the number of historic job versions that
   are kept.
 
+- `oidc_issuer` `(string: "")` - Specifies the Issuer URL for [Workload
```

Comment:
Do we want to give an example of the form that Issuer URL is supposed to take? Especially with the next sentence of the text showing the path component of the well-known endpoint, this could be confusing. Something like

> Specifies the Issuer URL for Workload Identity JWTs, which should include only the scheme and authority (and optionally port). For example, "https://nomad.example.com".


I'm also not sure if the spec cares about the trailing slash? Some of the issues in your test code in this PR have it and half don't. But if that's important, we should include a note about that here too. (I suspect _Nomad_ doesn't care with our HTTP router, but third-parties might.)

Reply:
> Do we want to give an example of the form that Issuer URL is supposed to take?

Good call. Actually using all of this plumbing is going to take a Tutorial, so I was trying to be minimal with the parameter doc, but an example is always a good idea.

> I'm also not sure if the spec cares about the trailing slash? ... **but third-parties might.**

Emphasis mine. AWS's IAM OIDC Identity Provider works either way, but who knows about others. Nomad itself doesn't. (Nomad doesn't actually use this value for anything functionally other than "is it set or not"!)

HTTP routing ignores this because a user may configure their proxy and issuer to be something like: `https://example.com/nomad`. However there's no need for Nomad to use the `/nomad` prefix for its OIDC Discovery endpoint: the proxy can translate the paths.

> Some of the issuers in your test code in this PR have it and half don't.

:facepalm: Fixing...

Looks like [Terraform](https://app.terraform.io/.well-known/openid-configuration) and [Google](https://accounts.google.com/.well-known/openid-configuration) both omit the trailing slash, so that's good enough for me!